src/components/LoginRegisterPanel/LoginRegisterPanel.tsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from 'react';
+import { IoCloseCircleOutline } from 'react-icons/io5';
+import { FaRegEye } from 'react-icons/fa';
+import { FaRegEyeSlash } from 'react-icons/fa';
+import { useNavigate } from 'react-router-dom';
+
+import styles from './LoginRegisterPanel.module.css';
+const LoginRegisterPanel = () => {
+  const [username, setUsername] = useState<string>('');
```

Comment:
If you add some initial value to use state - you don't have to add type between - typescript can infer type base on given initial value